### PR TITLE
Update mixer to use the latest version of the bt cache

### PR DESCRIPTION
**Pull request by pulkit-s**
We rebuilt caches to account for cl/478072824 and cl/477658325. Updating mixer to point to these latest versions.

### CODE REVIEW

**Review by pradh — COMMENTED** [review]
Do the other versions need to change too?

This might result in a bunch of diffs?

**Comment by pulkit-s** [comment]
> 

Done

**Comment by pradh** [comment]
Adding more folks for more pairs of eyes on diffs

File: internal/server/v0/triple/golden/get_triples/stat_var.json [hunk 1]
```diff
@@ -1129,149 +1129,6 @@
         "StatisticalVariable"
       ],
       "provenanceId": "dc/d7tbsb1"
-    },
-    {
-      "subjectId": "dc/06w4nhpt1vz26",
```
Comment (pradh) [inline internal/server/v0/triple/golden/get_triples/stat_var.json:1134]: Not sure about this too

File: internal/server/statvar/golden/get_statvar_group_node/demographics.json [hunk 2]
```diff
@@ -135,12 +135,6 @@
       "displayName": "Person by Earnings",
       "descendentStatVarCount": 139
     },
-    {
```
[the remark this answers is not in the document]
Comment (pradh) [reply internal/server/statvar/golden/get_statvar_group_node/demographics.json:138]: I think educational attainment still shows up under Education as the vertical spec is configured:
http://google3/datacommons/config/vertical_spec.textproto;l=476-481;rcl=478603844

File: internal/server/v1/triples/golden/bulk_triples_in/result1.json [hunk 3]
```diff
@@ -6848,83 +6848,6 @@
               ],
               "dcid": "Percent_WorkRelatedPhysicalActivity_NoActivityOrLightActivity_In_Count_Person",
               "provenanceId": "dc/d7tbsb1"
-            },
-            {
-              "types": [
-                "StatisticalVariable"
-              ],
-              "dcid": "dc/06w4nhpt1vz26",
-              "provenanceId": "dc/6zzrcr2"
```
[the remark this answers is not in the document]
Comment (pradh) [reply internal/server/v1/triples/golden/bulk_triples_in/result1.json:6857]: These are due to cl/472540248.  I think it is expected that certain auto-generated SVs from Pop/Obs datasets which have curated SVs will be removed. I spot checked a couple and these don't have SVObs.  e.g., dc/64exg7jchnekb is Count_CriminalActivities_MurderAndNonNegligentManslaughter

@Spaceenter for confirmation.

Note: even though the CL went in a month ago, the Biomed cache still had it, and now that we are building it past the change, we see these diffs.

File: internal/server/v0/triple/golden/get_triples/stat_var.json (hunk 1 above)
Comment (pradh) [reply internal/server/v0/triple/golden/get_triples/stat_var.json:1134]: Ditto. This is Count_Product_MobileCellularSubscription_AsFractionOf_Count_Person

**Review by pradh — APPROVED** [review]

File: internal/server/statvar/golden/get_statvar_group_node/demographics.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (beets) [reply internal/server/statvar/golden/get_statvar_group_node/demographics.json:138]: Got it. Perhaps the reason this only got removed now is the same as the biomed cache update in the [earlier comment](https://github.com/datacommonsorg/mixer/pull/935#discussion_r988404857).

**Review by beets — APPROVED** [review]

**Pull request merged**